WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsTrackerWrapper.kt

```diff
@@ -60,5 +60,13 @@ class AnalyticsTrackerWrapper
         AnalyticsUtils.trackWithSiteDetails(stat, siteModel)
     }
 
+    fun trackWithSiteDetails(
+        stat: Stat,
+        siteModel: SiteModel?,
+        properties: Map<String, Any>
+    ) {
+        AnalyticsUtils.trackWithSiteDetails(stat, siteModel, properties)
```

Comment:
We can pass `this` as a 1st parameter to [avoid creating a new instance of the wrapper](https://github.com/wordpress-mobile/WordPress-Android/pull/15465/files#diff-09531fb734dc98890b1902c4bc1ac651ff4a602c4de00a079fe08752a0b5aa2bR284).

Reply:
Hmm, not sure I follow there Antonis 🤔. Can you rephrase or elaborate? Thanks!

Reply:
Sorry for the cryptic comment 😄 
I'll try to elaborate. If I'm not missing something we can change the following call
`AnalyticsUtils.trackWithSiteDetails(stat, siteModel, properties)` to
`AnalyticsUtils.trackWithSiteDetails(this, stat, siteModel, properties)` and directly call the [modified trackWithSiteDetails method](https://github.com/wordpress-mobile/WordPress-Android/pull/15465/files#diff-09531fb734dc98890b1902c4bc1ac651ff4a602c4de00a079fe08752a0b5aa2bR243) without creating a [new instance of the `AnalyticsTrackerWrapper `](https://github.com/wordpress-mobile/WordPress-Android/pull/15465/files#diff-09531fb734dc98890b1902c4bc1ac651ff4a602c4de00a079fe08752a0b5aa2bR284)

Reply:
Ooooh, I see now, apologies. I was getting caught up in the fact that the [simpler variant](https://github.com/wordpress-mobile/WordPress-Android/blob/96b3eecd3145f9f5463620406507c2ef82d1e620/WordPress/src/main/java/org/wordpress/android/util/analytics/AnalyticsUtils.java#L281-L285) is used elsewhere anyway and we wouldn't be able to remove the in-place instantiation of AnalyticsTrackerWrapper, but now I understand better what you mean:
the call to AnalyticsUtils.trackWithSideDetails from inside the AnalyticsTrackerWrapper doesn't (or better, shouldn't) lead to creating a new AnalyticsTrackerWrapper.

Made the change with 25e4dd4.

Reply:
Thank you Stefanos 🙇